Say-Their-Name/say-their-names-web
# Added contribution footer for github and slack links.

#### What does this PR do?
- Adds the contribution items of slack and github links to the footer of the site

#### Description of Task to be completed?
- Not sure on this one but I believe it only needs merging

#### How should this be manually tested?
- run the site locally and visually check the footer is OK. Run react tests with the test:all script.

#### What are the relevant Trello Card stories? [ Trello Card Item ](https://trello.com/b/8x3mo2V5/say-their-names-web)
- Getting board not found - This was from a feature idea raised by myself here: https://github.com/Say-Their-Name/say-their-names-web/issues/154

#### Pull Request Type (check one)

#### Developer Checklist


#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12662269/90278823-f6143400-de5f-11ea-9d72-d67b3fd74ec1.png)
